src/utils/neynar/openframes/neynarValidator.ts

```diff
@@ -0,0 +1,54 @@
+import { OpenFramesRequest, RequestValidator, ValidationResponse } from '@open-frames/types';
+import { FarcasterOpenFramesRequest, FrameRequest, FrameValidationData } from '../../../core/types';
+import { NEYNAR_DEFAULT_API_KEY, neynarFrameValidation } from '../frame/neynarFrameFunctions';
+import { Options } from './types';
+
```

Comment:
Oh side note, we prefer Functional code vs Object Oriented, as it is much lighter in terms of memory and also easier to write test for.

Reply:
Fair enough. All I really care about is having something that satisfies a common interface so that developers can build their own request validators and have confidence they will be able to integrate.

Whether the implementation of the interface is a module, an object, or a class is up to the implementer.